src/PrivateDns/PrivateDns/help/Get-AzPrivateDnsZone.md

```diff
@@ -0,0 +1,171 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.PrivateDns.dll-Help.xml
+Module Name: Az.PrivateDns
+online version: https://docs.microsoft.com/en-us/powershell/module/az.privatedns/get-azprivatednszone
+schema: 2.0.0
+---
+
+# Get-AzPrivateDnsZone
+
+## SYNOPSIS
+Gets a Private DNS zone.
+
+## SYNTAX
+
+```
+Get-AzPrivateDnsZone [-ResourceGroupName <String>] [-Name <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzPrivateDnsZone** cmdlet gets a Private Domain Name System (DNS) zone from the specified resource group.
+If you specify the *Name* parameter, a single **PrivateDnsZone** object is returned.
+If you do not specify the *Name* parameter, an array containing all of the zones in the specified resource group is returned.
+You can use the **PrivateDnsZone** object to update the zone, for example you can add **RecordSet** objects to it.
+
+## EXAMPLES
+
+### Example 1: Get a zone
+```
+PS C:\> $Zone = Get-AzPrivateDnsZone -ResourceGroupName "MyResourceGroup" -Name "myzone.com"
+
+This example gets the Private DNS zone named myzone.com from the specified resource group, and then stores it in the $Zone variable.
+$Zone looks something like this: 
+
+Name                          : myzone.com
+ResourceId:                   : "/subscriptions/0e5a46b1-de0b-4ec3-a5d7-dda908b4e076/resourceGroups/MyResourceGroup/PrivateZones/myzone.com"
```

Comment:
For all of these examples, please remove your subscription id (replace with xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx). Same with Etag I would think.

Reply:
changed it.